script/Ciel/Day 13/13_01_02_CIEL13_7F.txt

```diff
@@ -785,7 +785,7 @@ I think... that'd be like coming across a rainbow in the night."
 [sha:ddfa02d33a06222e4bce6eef67be8c9b0369e3aa]{
 -- Page 456, Offset 36583.
 -- 起きてはほしいけど、起きるはずのない出来事をそう言っていたんです。
-It refers to something you want to happen, but that never will.
+It refers to something that you want to happen, but that never will.
```

Comment:
I think this was what I had in mind?
But it's not a big deal.
>...something you want to happen, which never will.

Reply:
Don't have any strong feelings but I prefer the conjunction to be a contrasting one like the 'but' currently used.

Reply:
Yeah I feel like it works better with the but